mkosi/resources/mkosi-tools/mkosi.conf.d/10-debian-ubuntu/mkosi.conf.d/ubuntu-keyring.conf

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=|!debian
+Release=|!bookworm
```

Comment:
`ubuntu-keyring` is also not available in bullseye, but Debian bullseye would match this, because (Debian != Debian) or (bullseye != bookworm) is true.

Reply:
So bullseye is missing a ton of packages just like Focal, so I don't think we can easily support `mkosi dependencies` on it.